thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js

```diff
@@ -94,86 +112,107 @@ export default Component.extend({
 
   /**
    * Anomaly feedback status
+   * @type {string}
    */
   status: reads('anomaly.attributes.status.firstObject'),
 
+  /**
+   * Anomaly baseline as computed by anomaly function
+   * @type {Float}
+   */
+  baseline: reads('anomaly.attributes.baseline.firstObject'),
+
   /**
    * Anomaly unique identifier
+   * @type {string}
    */
   anomalyId: computed('anomaly', function () {
     return get(this, 'anomaly').urn.split(':')[3];
   }),
 
   /**
-   * Anomaly duration
+   * Anomaly issue type for grouped anomaly (not supported yet)
+   * @type {string}
    */
-  duration: computed('anomaly', function () {
-    const anomaly = get(this, 'anomaly');
-    return moment.duration(anomaly.end - anomaly.start).humanize();
-  }),
+  issueType: null, // TODO
 
   /**
-   * Anomaly metric dimensions/filters
+   * Formatted metric label of anomaly topic metric
+   * @type {string}
    */
-  dimensions: computed('anomaly', function () {
-    const attr = get(this, 'anomaly').attributes;
-    const dimNames = attr.dimensions || [];
-    const dimValues = dimNames.reduce((agg, dimName) => { agg[dimName] = attr[dimName][0]; return agg; }, {});
-    return dimNames.sort().map(dimName => dimValues[dimName]).join(', ');
-  }),
+  metricLabel: computed(
+    'anomalyUrns',
+    'entities',
+    function () {
+      const { anomalyUrns, entities } = getProperties(this, 'anomalyUrns', 'entities');
 
-  issueType: null, // TODO
+      const metricUrns = filterPrefix(anomalyUrns, 'thirdeye:metric:');
 
-  metricFormatted: computed(
-    'dataset',
-    'metric',
-    function () {
-      const { dataset, metric } =
-        getProperties(this, 'dataset', 'metric');
-      return `${dataset}::${metric}`;
-    }
-  ),
+      if (!metricUrns) { return; }
 
-  dimensionsFormatted: computed('anomaly', function () {
-    const dimensions = get(this, 'dimensions');
-    return dimensions ? ` (${dimensions})` : '';
-  }),
+      const metricUrn = metricUrns[0];
 
-  changeFormatted: computed('change', function () {
-    const change = get(this, 'change');
-    const prefix = change > 0 ? '+' : '';
-    return `${prefix}${humanizeFloat(change * 100)}%`;
-  }),
+      return toMetricLabel(metricUrn, entities);
+    }
+  ),
 
+  /**
+   * Formatted anomaly start time
+   * @type {string}
+   */
   startFormatted: computed('anomaly', function () {
     return moment(get(this, 'anomaly').start).format('MMM D YYYY, hh:mm a');
   }),
 
+  /**
+   * Formatted anomaly end time
+   * @type {string}
+   */
   endFormatted: computed('anomaly', function () {
     return moment(get(this, 'anomaly').end).format('MMM D YYYY, hh:mm a');
   }),
 
+  /**
+   * Formatted anomaly duration
+   * @type {string}
+   */
+  durationFormatted: computed('anomaly', function () {
+    const anomaly = get(this, 'anomaly');
+    return moment.duration(anomaly.end - anomaly.start).humanize();
+  }),
+
   //
   // expand / hide anomaly header
   //
```

Comment:
I know this is not part of the PR, but is this supposed to be a comment for something? If not, let's remove it.

Reply:
fixed